test/miscellaneous.js

```diff
@@ -6,7 +6,7 @@ var should = require('should');
 describe('Miscellaneous', function() {
   it('NUMERAL', function() {
     misc.NUMERAL(10000, '0,0.0000').should.equal("10,000.0000");
-    misc.NUMERAL(10000.23, '0,0').should.equal("10,000");
+    misc.NUMERAL(10000.23, { mantissa: 0, thousandSeparated: true }).should.equal("10,000");
```

Comment:
Does that mean we can't achieve this format with string anymore?

Reply:
I found something about this here: https://github.com/BenjaminVanRyseghem/numbro/issues/333.
Maybe we can replace it internally, in a definition of `NUMERAL`.

http://numbrojs.com/news.html#release-2-0-0
> Hopefully, numbro 2.0.0 is 99% backward compatible with the previous version (1.11.1).

We found incompatible 1% :)